server/controllers/configurationsController/configurationsController.test.js

```diff
@@ -161,22 +187,35 @@ describe('ConfigurationsController', () => {
           data: result,
         })
       })
-    })
 
-    describe('When unsuccessful', () => {
-      it('returns a status of 400 and an error message', async () => {
-        const request = createRequestWithUser(params)
+      it('returns a status of 200 and an array of only published/active configurations', async () => {
+        const result = [
+          createConfiguration({
+            ...activeConfig,
+            owner_display_name: 'Owl Owlson',
+          }),
+          createConfiguration({
+            ...activeConfig2,
+            owner_display_name: 'Eagle Eagleson',
+          }),
+          {
```

Comment:
fixture function

Reply:
The fixture function adds the status property to the configuration, and this one is replicating a legacy configuration where the status property is non existent. This better represents the different documents we are bound to come in contact with.